bin/cpumask.py

```diff
@@ -27,6 +27,14 @@ def process_options():
     myglobal.args = parser.parse_args()
 
 
+def linux_hexmask(hexmask):
```

Comment:
"""a brief description for this function helps"""

Reply:
good point